src/core/cgroup.c

```diff
@@ -2584,6 +2584,73 @@ int unit_get_ip_accounting(
         return r;
 }
 
+/* The below is borrowed from cgtop/cgtop.c */
```

Comment:
You should not just take a long block of code like this and copy it from one file to another. Instead, this needs to be turned into a function somewhere in `src/shared` and called from both locations.